pdns/recursordist/docs/settings.rst

```diff
@@ -513,11 +513,16 @@ That is, only if both the limits apply, the record will not be cached.
 
 ``ecs-minimum-ttl-override``
 ----------------------------
+.. versionchanged:: 4.5.0
+  Old versions used default 0.
+
 -  Integer
--  Default: 0 (disabled)
+-  Default: 1
 
 This setting artificially raises the TTLs of records in the ANSWER section of ECS-specific answers to be at least this long.
-While this is a gross hack, and violates RFCs, under conditions of DoS, it may enable you to continue serving your customers.
+Setting this to a value greater than 1 technically is an RFC violation, but might improve performance a lot.
+Using a value of 0 impacts performance of TTL 0 records greatly, since it forces the recursor to contact
+authoritative servers each time a client requests them.
```

Comment:
```suggestion
authoritative servers every time a client requests them.
```

Reply:
afaik (as a non-native speaker) both phrases are ok and equivalent. Any particular reason you prefer "every time"?

Reply:
I _think_ "each time" implies less a rule than a repetition of individual events. But I'm fine with the original phrasing, otherwise I guess we need a native speaker :)